packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -509,20 +509,18 @@ contract VerifyOPCM is Script {
             // All getters must be accounted for in expectedGetters mapping
             if (bytes(verificationMethod).length == 0) {
                 console.log("ERROR: Getter '%s' is not accounted for in expectedGetters mapping", functionName);
-                return false;
+                success = false;
+                continue;
```

Comment:
Why did we decide to standardize on `success = false` rather than standardize on `return false`? 

What are the tradeoffs between those two options?

Reply:
We went with success = false (continue checking) over return false (fail fast) because:
Better diagnostics: You get a complete report of ALL issues in one run instead of discovering problems one at a time.
Consistent with the design: The main verification loop already uses this pattern.

When we DO use return false: Only for fatal structural errors where continuing doesn't make sense (like bytecode length mismatches).

If 3 out of 15 OPCM getters are misconfigured, you see all 3 problems at once rather than fixing them one by one across multiple test runs.